src/styles/01-atoms/_images/image/_image.scss

```diff
@@ -1,18 +1,30 @@
 @mixin image {
-  display: block;
-  height: auto;
-  max-width: 100%;
+    display: block;
+    height: auto;
+    max-width: 100%;
 }
 
 img,
 picture {
-  @include image;
+    @include image;
 }
 
 figure {
-  margin: 0; //override normalize
+    margin: 0; //override normalize
 }
 
 .figure__caption {
-  font-style: italic;
+    font-style: italic;
+}
+
+.annotation__image {
```

Comment:
Can we use the `.figcaption` class above instead of the `.annotation__image` class you've added? It'll better describe the element we're styling once you've made the requested changes in `Image.tsx`